README.md

```diff
@@ -1,40 +1,104 @@
-To develop on RAMS Core:
+# Getting Started Developing in RAMS: Community Edition
 ===
 
-1. Clone the quickstart repo.
-2. `git clone https://github.com/RAMSProject/rams.git src/uber`
-3. Clone any other plugins you are using into src/
-4. `docker-compose -d up`
-5. Access via your local path output of `docker-machine ip dev`) [TODO: elaborate]
-6. Edit the code in src/ - most updates should be live. Use `docker-compose` to restart daemon if needed.
+## Getting to Know RAMS: What We Use
+ If you just want to spin up an instance, feel free to skip down to "Before You Start: Setting Up Your Environment." However, if you're brand new, you should familiarize yourself with our structure.
 
-If you need to build/change Python or Javascript dependencies you'll have to change `requirements.txt` or `bower.json` and manually rebuild the ramsproject/rams container locally. Ditto if you're changing something in Sideboard itself. Note: if you don't know if you're doing either of these, you probably aren't.
+### RAMS Technology
 
+ RAMS is based on **Sideboard**, a custom framework that ties plugins together and runs them as a web app. The "uber" plugin - [which is its own repo](https://github.com/magfest/ubersystem) - is itself a plugin for Sideboard. In general you shouldn't need to worry about Sideboard, but be aware that it has several overrides for built-in Python libraries. It will be automatically included when you install RAMS following the instructions below.
```

Comment:
This should point at our copy, because the uber plugin is the ramsproject/rams repo - it hasn't been renamed because the name "uber" is hardcoded a bunch of places.